src/Components/Button.js

```diff
@@ -1,23 +1,34 @@
 import React, { useState } from 'react';
 import * as Icons from '../Components/Icons';
 import classes from '../App.module.css';
+require('typeface-inter');
 
 const Button = (props) => {
 
+	console.log('this is a button', props)
+
 	let svg;
-	if(props.icon == 'tag') {
-		svg = <Icons.Tag />
+	if(props.icon == 'command') {
```

Comment:
Instead of this complicated else if structure, you can just create an object matching names to svg's, and it'll also be faster (since looking up something in a hashtable--which is essentially what an object is--is faster than stepping through a bunch of if statements sequentially)

```
const icons = {
  bookmark: <Icons.Bookmark />,
  plus: <Icons.Plus />,
  ...
}

svg = icons[props.icon];
```